EXOSIMS/Scripts/sampleScript_occulter.json

```diff
@@ -1,20 +1,20 @@
 {
```

Comment:
Please revert this sample script to its original contents.  If you want, you can add your own sampleScript for your specific use case.

Reply:
fixed